intake_esm/core.py

```diff
@@ -8,7 +8,7 @@
 import intake
 import numpy as np
 import pandas as pd
-from tqdm import tqdm
+from tqdm.auto import tqdm
```

Comment:
How about wrapping the imports in a try block so that when `ipywidgets` is not installed, we revert back to the default `tqdm`

```python
try:
    import ipywidgets
    from tqdm.auto import tqdm
except ImportError as e:
    from tqdm import tqdm
```